modules/use-default-security-policies.adoc

```diff
@@ -34,6 +34,11 @@ The default policies have preconfigured parameters and belong to categories such
 * System Modification
 * Vulnerability Management
 
-You can edit these categories create your own categories.
+You can edit these categories and create your own categories.
```

Comment:
This is this still not clear to me. Is it
You can edit these categories _**to**_ create your own categories.
You can edit these categories _**or**_ create your own categories.